### DISCUSSION TITLE
A major update of stablediffusion-infinity has been released with a redesigned UI and better outpainting quality

### DISCUSSION THREAD

**Discussion by lkwq007** [lkwq007/stablediffusion-infinity#86] [Announcements]
New features

- better outpainting quality powered by runwayml stable diffusion inpainting model
- img2img support (not available for the first model choice in the UI)
- photometric correction for preserving existing contents/suppressing seams
- save/load/reset the canvas
- canvas size/selection size can be adjusted at the runtime
- better UI for uploading images (uploading multiple images/editing the images before putting them into the canvas)
- batch_size(sample number)/seed/scheduler support
- keyboard shortcut support
- local checkpoint/model weights support (v1.4/v1.5 are supported)
- textual inversion embeddings support

https://user-images.githubusercontent.com/1665437/197317716-8500e7fb-6b1e-4ac5-8aa0-3338bd86831d.mp4

**Comment by leonnn1** [comment]
This is one of the best webuis out there, it's unique and I really amazing, I will make a tutorial on my channel about it cause it deservesm thank you!
